merger.js

```diff
@@ -49,28 +49,32 @@ function processDir(dir)
 }
 
 function processFile(file, include) {
-  if (file === mainFile && mainIsProcessed) {
+  let processedOnce = false;
+  for (let i = 0; i < processOnce.length; i++) {
+    if (processOnce[i] == file) {
+      // console.log("Found in processedOnce: ", file);
+      processedOnce = true;
+      break;
+    }
+  }
+ 
+ if (file == outputFile) return;
```

Comment:
This is a good one! I always put my outputfile in a different directory or used a different extension for it, but this quick check is perfect, I would just add it to the else if block below.